tests/dvm/darwinia/tests/config.ts

```diff
@@ -6,3 +6,6 @@ export const config = {
 	gas: 4000000,
 	jsontestAddress: "",
 };
+
+// EXTRINSIC_GAS_LIMIT = [BLOCK_GAS_LIMIT - BLOCK_GAS_LIMIT * (NORMAL_DISPATCH_RATIO - AVERAGE_ON_INITIALIZE_RATIO) - EXTRINSIC_BASE_Weight] / WEIGHT_PER_GAS  = (1_000_000_000_000 * 2 * (0.75-0.025) - 125_000_000) / 40_000
+export const EXTRINSIC_GAS_LIMIT = 36246875;
```

Comment:
Need review and double check.

Reply:
This constant is related to the `WEIGHT_PER_GAS`, If `WEIGHT_PER_GAS` is right, this value is right too. There is a test case designed for the calculation process.